src/db-schema.jl

```diff
@@ -557,54 +536,72 @@ function create_triggers(db::SQLite.DB)
         end;
         """,
         """
-          create trigger afterInsertInformedVote after insert on InformedVote 
-          when new.informed = 1 
-          begin
-             insert into InformedTally(
-                    tag_id
-                  , post_id
-                  , note_id
-                  , count
-                  , total
-              ) 
-              values (
-                  new.tag_id
-                  , new.post_id
-                  , new.note_id
-                  , (new.vote = 1)
-                  , (new.vote != 0)
-              ) 
-              on conflict(tag_id, post_id, note_id) do update
-              set
-                    count   = count + (new.vote = 1)
-                  , total   = total + (new.vote != 0)
-              ;
-          end;
+        create trigger afterInsertInformedVote after insert on InformedVote 
+        when new.informed = 1 
+        begin
+           insert into InformedTally(
+                  tag_id
+                , post_id
+                , note_id
+                , count
+                , total
+            ) 
+            values (
+                new.tag_id
+                , new.post_id
+                , new.note_id
+                , (new.vote = 1)
+                , (new.vote != 0)
```

Comment:
What does this do / how does sqlite handle boolean operations like this?

Intuitively, my expectation would be that this evaluates to:

```
| new.vote | count | total |
|----------|-------|-------|
|    1     |   1   |   1   |
|    0     |   0   |   0   |
|   -1     |   0   |   0   |
```

Is that correct and if so, what does it mean wrt to InformedTallies?

Reply:
Actually it will be:

| new.vote | count | total |
|----------|-------|-------|
|    1     |   1   |   1   |
|    0     |   0   |   0   |
|   -1     |   0   |   1   |

These values are added to the informed tally. That is, count is incremented if the vote is an upvote, and total is incremented if there is an upvote or a downvote (e.g. not a vote value of 0).